lib/openapi_parser/config.rb

```diff
@@ -19,6 +19,10 @@ def strict_response_validation
     @config.fetch(:strict_response_validation, false)
   end
 
+  def strict_reference_validation
+    @config.fetch(:strict_reference_validation, false)
```

Comment:
I think this is very good feature, so enabling it by default will make the most users happy.
But we shouldn't break backward compatibility...

I'd like to see the current version set to false, with a message saying like "this will be true in the future version, so if you don't want it to be changed, please explicitly set it to false" when it is not set.
(i.e. https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/option.rb#L34 )
By displaying it this way, users can update without breaking backward compatibility by setting the warning to not be displayed.
If you think it would be better to modify the wording in the README, please do that too.

Reply:
I updated the README and added a deprecation warning if the config key is not set.

This will also have the effect of showing a deprecation warning even for people who want the default to be `true`, but I think we need to do it to force everyone to consider their preferred behavior.

I also added a "request for comments" in the README -- if anyone actually wants it to be `false` by default, I would love to hear their use case (but I think `true` is generally better - as you and I both seem to agree).